genienlp/models/transformer_lstm.py

```diff
@@ -56,17 +58,28 @@ def __init__(self, config=None, *inputs, args, vocab_sets, tasks, save_directory
         encoder_embeddings = args.pretrained_model
         config = AutoConfig.from_pretrained(encoder_embeddings, cache_dir=args.embeddings)
         args.dimension = config.hidden_size
-        self.numericalizer = TransformerNumericalizer(encoder_embeddings,
-                                                      max_generative_vocab=args.max_generative_vocab,
-                                                      cache=args.embeddings,
-                                                      preprocess_special_tokens=args.preprocess_special_tokens)
+        self.numericalizer = TransformerNumericalizer(encoder_embeddings, args, max_generative_vocab=args.max_generative_vocab)
+        
+        self.numericalizer.get_tokenizer(save_directory)
         self.init_vocab_from_data(vocab_sets, tasks, save_directory)
 
         logger.info(f'Initializing encoder and decoder embeddings')
-        if save_directory is not None:
-            self.encoder_embeddings = AutoModel.from_config(config)
+        
+        if args.do_ner:
+            if type(config) == BertConfig:
```

Comment:
Should use `isinstance` here.

Reply:
Well I was but then realized`isinstance` checks for inheritance so `isinstance(AutoConfig.from_pretrained('xlm-roberta-base', cache_dir='.embeddings'), BertConfig)` is True!
I can reorder ifs to check for XLMRobertaConfig first, but using `type` is cleaner?